components/Pages/Payruns/CreatePayRunModal/AdHocAndReleases.js

```diff
@@ -3,23 +3,26 @@ import {
   Button,
   Container,
   DatePicker,
-  FormGroup,
+  FormGroup, Hint,
   HorizontalSeparator,
   RadioGroup,
   VerticalSeparator,
 } from 'components';
-import React from 'react';
+import React, { useMemo } from 'react';
 import { Controller, useForm } from 'react-hook-form';
 import { formValidationSchema } from 'service/formValidationSchema';
+import { addDays, differenceInDays } from 'date-fns';
+import { useLatestPayRunToDate } from '../../../../api';
+
```

Comment:
can be shorthanded as 'api';

remove empty 16 line

Reply:
Updated